ci-operator/step-registry/install-operators/install-operators-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: install-operators
+  from_image:
```

Comment:
I found an imagestream named cli-jq

I'm wondering if this will satisfy this need rather than having us maintain this
https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/k8s/ns/ocp/imagestreamtags/cli-jq%3Alatest

It seem like we could just use
```
from_image:
    namespace: ocp
    name: cli-jq
    tag: cli
```

Reply:
I'd rather do that. Let me give it a shot real quick.

Reply:
@ascerra that worked perfectly. I will remove the work we have done from this PR. I'll keep that repo around for now though. 

CC: @ryankwilliams